vllm/distributed/kv_transfer/kv_connector/v1/nixl_connector.py

```diff
@@ -62,6 +92,8 @@ class ReqMeta:
     remote_host: str
     remote_port: int
     remote_engine_id: str
+    do_remote_prefill: bool = False
```

Comment:
would you mind explaining to me what `do_remote_prefill=True` means?

Reply:
If this attribute is True, it means this is a decode request and its prefill computation has been done remotely (also some remote kv may need to be fetched to local before decode computation).